src/order/conditionally_complete_lattice.lean

```diff
@@ -1002,12 +1002,16 @@ noncomputable instance with_bot.conditionally_complete_lattice
   ..with_bot.has_Sup,
   ..with_bot.has_Inf }
 
-/-- Adding a bottom and a top to a conditionally complete lattice gives a bounded lattice-/
-noncomputable instance with_top.with_bot.bounded_lattice {α : Type*}
-  [conditionally_complete_lattice α] : bounded_lattice (with_top (with_bot α)) :=
+/-- Adding a bottom and a top to a conditionally complete lattice gives a bounded order-/
+instance with_top.with_bot.bounded_order {α : Type*}
+  [conditionally_complete_lattice α] : bounded_order (with_top (with_bot α)) :=
```

Comment:
Nuke this instance. It's not using `[conditionally_complete_lattice α]`. The correct one is
```
instance with_top.bounded_order {α : Type*} [order_bot α] : bounded_order (with_top α) :=
{ ..with_top.order_bot,
  ..with_top.order_top }
```